.changes/unreleased/tinymce-TINY-10916-2024-05-15.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Fixed a bug where transparent colors were converted to black.
```

I think since the category is already Fixed it nice if we can avoid using fixed in the changelog message.

```suggestion
body: CSS color values set to `transparent` were incorrectly converted to '#000000`.
```